### PR TITLE
Discoverpage

### PR DESCRIPTION
**Pull request by christophschmitz**
Discover page of Moviescout

[Deployment link](https://review-discoverpage-eayzyyh4rp.herokuapp.com/discover)

### CODE REVIEW

**Review by marwinburesch — CHANGES_REQUESTED** [review]
Thank you very much ☔

File: src/app/pages/Discover/Discover.tsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        <Header withBackButton isHighlighted>
+          Discover
+        </Header>
+        <div className={styles.tagGroup}>
+          <TagGroup tagList={mockTags} />
+        </div>
+      </section>
+      <section className={styles.cardWrapper}>
+        {mockMovies.length !== 0 &&
+          mockMovies.map((movie) => <Card key={movie.title} {...movie} />)}
```
Comment (marwinburesch) [inline src/app/pages/Discover/Discover.tsx:133]: `onBookmarkClick` should get assigned here inside the `.map`, aim for a `handleBookmarkClick`. We won't get this from our API later

**Review by marwinburesch — CHANGES_REQUESTED** [review]
Cheers! Just one more thing D:

File: src/app/pages/Discover/Discover.module.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+.topSection {
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  padding: 2.5rem 0 0 1rem;
+  align-items: center;
+}
+
+.cardWrapper {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
```
Comment (marwinburesch) [inline src/app/pages/Discover/Discover.module.css:22]: Let's have this be responsive. It should be `1fr` on iPhone 5 and then scale up to have two or more rows if it fits

**Review by marwinburesch — CHANGES_REQUESTED** [review]
Cheers 🐘

**Review by phlgr — APPROVED** [review]
LGTM 🎆

**Review by marwinburesch — APPROVED** [review]

**Pull request merged**